src/request/src/network.js

```diff
@@ -179,6 +179,32 @@ export class KinveyRequest extends NetworkRequest {
     this.trace = options.trace === true;
   }
 
+  static executeShort(options, client, dataOnly = false) {
```

Comment:
The name (and the very fact there are two so similar methods) is a work in progress, as is all of this PR. Just a place to discuss. Also, I wanted to test it out by using it for this feature.

I think there should be one shorthand method and I think it should be called `execute()` and have the defaults of the current `executeShort()`. Or others, based on what you guys think is the most used case.

I really don't want to be building my own URLs using a library each time I want to make a request, especially if half of the URL is the same for each request. This is why I don't think the current `execute()` is enough of a convenience and would prefer to keep something like the current `executeShort()`. Because as is, `execute` only removes the "new ....; .execute()` part. And there is much more that can be omitted most of the time.